webapp/channels/src/components/integrations/abstract_oauth_app.test.tsx

```diff
@@ -85,12 +90,12 @@ describe('components/integrations/AbstractOAuthApp', () => {
         const wrapper = shallow(
             <AbstractOAuthApp {...props}/>,
         );
-
-        wrapper.instance().updateName({target: {value: 'new name'}});
+        const evt = {preventDefault: jest.fn(), target: {value: 'new name'}} as any;
```

Comment:
instead of as `any` I would prefer casting to correct expected event
```suggestion
        const evt = {preventDefault: jest.fn(), target: {value: 'new name'}} as unknown as ChangeEvent<HTMLInputElement>;
```

This suggestion would apply for all `as any` usage in this test file.